### PR TITLE
[MAINT] add consortium authorship

### LINKED ISSUE #3805: [DOC] order of contributors
Adding the citation.cff allows to "cite" nilearn from the github UI in bibtex or APA format. 
See below.

By default it will list contributors in the order they appear in the citation.cff file: so alphabetically for now.

Opening an issue to discuss if we want a different order and if so which one.

```bibtex
@software{Chamma_nilearn,
author = {Chamma, Ahmad and Frau-Pascual, Aina and Rothberg, Alex and Abadie, Alexandre and Abraham, Alexandre and Gramfort, Alexandre and Savio, Alexandre and Thual, Alexis and Kanaan, Amadeus and Pinho, Ana Luisa and Idrobo, Andrés Hoyos and Kieslinger, Anne-Sophie and Rokem, Ariel and Mensch, Arthur and Duran, Audrey and Cipollini, Ben and Thirion, Bertrand and Nguyen, Binh and Cakan, Caglar and Gorgolewski, Chris and Markiewicz, Chris and Horea, Christian and Gerloff, Christian and Reininger, Colin and Lane, Connor and Delettre, Céline and Gale, Dan and Gomez, Daniel and Bzdok, Danilo and Ellis, David G and Wassermann, Demian and Pisner, Derek and Orfanos, Dimitri Papadopoulos and DuPre, Elizabeth and Dohmatob, Elvis and Larson, Eric and Edmond, Evan and Pedregosa, Fabian and Liem, Franz and Varoquaux, Gael and Hollander, Gilles de and Kiar, Greg and Gilmore, Greydon and Lemaitre, Guillaume and Wang, Hao-Ting and Aggarwal, Himanshu and Abenes, Ian and Vogel, Jake and Margeta, Jan and Grobler, Jaques and Gors, Jason and Rasero, Javier and Kossaifi, Jean and King, Jean-Rémi and Dalenberg, Jelle Roelof and Lefort-Besnard, Jeremy and Dockes, Jerome and Chevalier, Jerome-Alexis and Wiesner, Johannes and Gorrono, Jon Haitz Legarreta and Sassenhagen, Jona and Teves, Joshua and Huntenburg, Julia and Peraza, Julio A and Daddy, Kamalakar Reddy and Sitek, Kevin and Helwegen, Koen and Shmelkov, Konstantin and Chawla, Kshitij and CHEN, Kun and Sasse, Leonard and Estève, Loic and Tetrel, Loic and Paz, Luz and Pietrantoni, Manon and Perez-Guevara, Martin and Wegrzyn, Martin and Goncalves, Mathias and Ekman, Matthias and Joulot, Matthieu and Rahim, Mehdi and Eickenberg, Michael and Hanke, Michael and Notter, Michael and Waskom, Michael and Wang, Michelle and Torabi, Mohammad and Boos, Moritz and Song, Myeong Seop and Clarke, Natasha and Shah, Neelay and Gensollen, Nicolas and Esteban, Oscar and Bogdan, Paul and Sanz-Leon, Paula and Herholz, Peer and Gervais, Philippe and Bellec, Pierre and Glaser, Pierre and Quirion, Pierre-Olivier and Raamana, Pradeep Reddy and Meudec, Raphael and Luke, Robert and Williamson, Robert and Guidotti, Roberto and Phlypo, Ronald and Hammonds, Ryan and Gau, Rémi and Patalasingh, Sachin and Hahn, Sage and Bougacha, Salma and Johnson, Sam Buck and Jawhar, Sami and Steinkamp, Simon and Singh, Sourav and Meisler, Steven and Lan, Sylvain and Takerkart, Sylvain and Samanta, Tarun and Salo, Taylor and Bazeille, Thomas and Vanasse, Tom and Diogo, Vasco and Michel, Vincent and Fritsch, Virgile and Halchenko, Yaroslav and Mzayek, Yasmin and Baratz, Zvi and Nájera, Óscar},
license = {BSD-4-Clause},
title = {{nilearn}},
url = {https://github.com/nilearn/nilearn}
}
```

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes #3805 

Changes proposed in this pull request:

- makes sure that scripts in charge of citation.cff curation always keep the consortium author at the top of the list of authors in the citation.cff.

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -137,7 +137,7 @@ Some other past or present contributors are:
 * `Maximilian Cosmo Sitter`_
 * `Mehdi Rahim`_: Air Liquide, France
 * `Michael Eickenberg`_: Flatiron Institute, New-York, New-York, USA
-* `Michael Hanke`_: Psycho­informatics, Forschungszentrum Jülich GmbH, Jülich, Germany
```
Comment (Remi-Gau) [inline AUTHORS.rst:140]: Removes a character with some "weird" encoding that made my VS code lose its mind

File: CITATION.cff [hunk 2]
```diff
@@ -27,6 +27,7 @@ contact:
 license: BSD-4-Clause
 
 authors:
+  - family-names: Nilearn contributors
```
Comment (Remi-Gau) [inline CITATION.cff:30]: @emdupre I went with the first thing that crossed my mind but happy to use anything else.

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Review by ymzayek — APPROVED** [review]
LGTM

File: CITATION.cff (hunk 2 above)
Comment (emdupre) [reply CITATION.cff:30]: I think that's fine ! I'd say "developers", but I don't think this word choice is going to make or break the consortium idea :)

**Review by emdupre — APPROVED** [review]
Thx @Remi-Gau !

File: CITATION.cff (hunk 2 above)
Comment (Remi-Gau) [reply CITATION.cff:30]: I don't think so either. And we still have some time before the release in case we got a change of heart

**Pull request merged**